src/net/sourceforge/kolmafia/preferences/Preferences.java

```diff
@@ -359,6 +359,7 @@ public class Preferences {
         "latteModifier",
         "latteUnlocks",
         "leafletCompleted",
+        "ledCandleDropped",
```

Comment:
Do we know the "on Ascension" status of map usage and drops?

Reply:
map usage / drops is a daily pref, so it gets reset on ascension automatically

Reply:
OK.  I didn't know that, in general, and the presence of _shortOrderCookCharge in the Ascension resets made me think otherwise.  Presumably _shortOrderCookCharge in that list is redundant?